resources/backpack_role.yaml

```diff
@@ -107,6 +107,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - coordination.k8s.io
```

Comment:
curious why is this needed ?

Reply:
It was not successfully running the node collection due to an api restriction. Adding the coordination api group resolved it.